### ISSUE TITLE
Optimization Level Setting Not Detectable By Screen Readers

### ISSUE THREAD

**Issue by joejoe04** [wp-media/imagify-plugin#651]
**Describe the bug**
When viewing the plugin settings page using a screen reader like JAWS or NVDA, the Optimization Level settings are not detectable and so they cannot be changed.

**To Reproduce**
1. Log in to WordPress site with Imagify installed using a screen reader program
2. Navigate to the Imagify plugin settings page
3. The Optimization Level setting will not be detectable

**Expected behavior**
Screen reader programs should be able to detect the Optimization Level settings so they can be changed.

**Additional context**
Here is the ticket where this was reported to us by a user:
https://secure.helpscout.net/conversation/1750760770/318055/

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by GeekPress** [comment]
Closed by #664
